pkg/utils/pci_allocator.go

```diff
@@ -0,0 +1,83 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/containernetworking/plugins/pkg/ns"
+	"os"
+	"path/filepath"
+)
+
+type PCIAllocation interface {
+	SaveAllocatedPCI(string, string) error
+	CleanAllocatedPCI(string) error
+	IsAllocated(string) error
+}
+
+type PCIAllocator struct {
+	dataDir string
+}
+
+func NewPCIAllocator(dataDir string) PCIAllocation {
```

Comment:
m(iiinor)nit: docstring

Reply:
this method should IMO return *PCIAllocator

the caller should store it in a PCIAllocation interface